test_manager/features/environment.py

```diff
@@ -43,6 +43,10 @@
 JAVA_TA_PATH = "/test_agent/java/target/tck-test-agent-java-jar-with-dependencies.jar"
 DISPATCHER_PATH = "/dispatcher/dispatcher.py"
 
+repo = git.Repo('.', search_parent_directories=True)
```

Comment:
Why are we using git?

Reply:
Because it allows us to access the dispatcher in the root directory. We launch the dispatcher as a class instead of a process

Reply:
Do we have this dependency in requirements file? I feel it's not correct to use git dependency here. I remembered last time, @princemac did something to fix this. Will ask him to review it as well.

Reply:
It's a requirement in up-python requirements

Reply:
you are doing this to import the testmanager
sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
we can try to do the same or else do both in one way

Reply:
how about something like this? this allows us to do the imports straight from the root and only have to do it one way:

`repo = git.Repo(".", search_parent_directories=True)
sys.path.append(repo.working_tree_dir)
from test_manager.testmanager import TestManager
from dispatcher.dispatcher import Dispatcher
from test_manager.features.utils import loggerutils`

Reply:
What If I want to test only java , cpp and rust sdk , in that case I won't be having up-python

Reply:
you would still need to install requirements.txt since test-manager relies on dependencies in there

Reply:
the same requirements.txt that's installing up-python is also installing behave. while that requirement is there, git will be installed